fontconfig.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: fontconfig
   version: 2.14.0
-  epoch: 0
+  epoch: 1
```

Comment:
epoch bump is only needed for python3 part.

Reply:
Thats great, thanks.  I'll update the PR now.